cli/azd/pkg/commands/pipeline/pipeline_manager.go

```diff
@@ -117,6 +127,23 @@ func (i *PipelineManager) preConfigureCheck(ctx context.Context, infraOptions pr
 		)
 	}
 
+	tenantId, err := i.subResolver.LookupTenant(ctx, i.Environment.GetSubscriptionId())
+	if err != nil {
+		return configurationWasUpdated, fmt.Errorf("getting tenant id for subscription %s. Error: %w",
+			i.Environment.GetSubscriptionId(), err)
```

Comment:
I would simply return `err` here. The underlying error messages already communicate enough details.